packages/cli/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@celo/celocli",
   "description": "CLI Tool for transacting with the Celo protocol",
-  "version": "1.2.1-dev",
+  "version": "1.2.2",
```

so looks like you released version 1.2.2, that means that the current dev version on master should be 1.2.3-dev (the dev version of a future release) if that makes sense
```suggestion
  "version": "1.2.3-dev",
```